CRM/Contribute/Import/Parser/Contribution.php

```diff
@@ -335,21 +335,19 @@ public function import($onDuplicate, &$values) {
 
           return CRM_Import_Parser::VALID;
         }
-        else {
-          $labels = [
-            'id' => 'Contribution ID',
-            'trxn_id' => 'Transaction ID',
-            'invoice_id' => 'Invoice ID',
-          ];
-          foreach ($dupeIds as $k => $v) {
-            if ($v) {
-              $errorMsg[] = "$labels[$k] $v";
-            }
-          }
-          $errorMsg = implode(' AND ', $errorMsg);
-          array_unshift($values, 'Matching Contribution record not found for ' . $errorMsg . '. Row was skipped.');
-          return CRM_Import_Parser::ERROR;
+        $labels = [
+          'id' => 'Contribution ID',
+          'trxn_id' => 'Transaction ID',
+          'invoice_id' => 'Invoice ID',
+        ];
+        foreach ($dupeIds as $k => $v) {
+          if ($v) {
+            $errorMsg[] = "$labels[$k] $v";
+          }
         }
+        $errorMsg = implode(' AND ', $errorMsg);
+        array_unshift($values, 'Matching Contribution record not found for ' . $errorMsg . '. Row was skipped.');
+        return CRM_Import_Parser::ERROR;
       }
     }
```

Comment:
This looks wonky in web view but 

![Uploading Screenshot from 2020-12-11 14-41-47.png…]()